contracts/src/interfaces/ITeleporterTokenBridge.sol

```diff
@@ -46,9 +44,11 @@ struct SendTokensInput {
  * @param recipientContract the contract on the destination chain that will be called
  * @param recipientPayload the payload that will be provided to the recipient contract on the destination chain
  * @param requiredGasLimit the required amount of gas needed to deliver the message on its destination chain,
- *                         including token operations and the call to the recipient contract.
+ * including token operations and the call to the recipient contract.
  * @param recipientGasLimit the amount of gas that will provided to the recipient contract on the destination chain,
- *                          which must be less than the requiredGasLimit of the message as a whole.
+ * which must be less than the requiredGasLimit of the message as a whole.
+ * @param multiHopFallback in the case of a multi-hop transfer, the address where the tokens
+ * are sent on the source chain if the transfer is unable to be routed to its final destination.
  * @param fallbackRecipient address where the bridged tokens are sent if the call to the recipient contract fails.
```

Comment:
Can we update the `fallbackRecipient` comment to specify that the funds are sent to that address on the chain identified by `destinationBlockchainID`?